### DISCUSSION TITLE
Width and Height Slide Options Not Working as Intended?

### DISCUSSION THREAD

**Discussion by bechnokid** [biati-digital/glightbox#399] [Q&A] [answered]
Apologies if this question has been asked many times, but I've been at a loss regarding how to adjust the width and height of a slide container, and I haven't seen any answers that resolve the current issue I'm having.

Neither `data-width="300px"` nor `data-glightbox='width: 300px'` seem to do anything for any slides I'm working on.

I'll provide some screenshots as examples. 

This is the original slide container.
![image](https://user-images.githubusercontent.com/16202812/213757785-7385634d-c499-4d9f-bb56-183a7ec4a0f6.png)

This is the size that I want the slide container to be, which can only be done by adjusting the CSS.
![image](https://user-images.githubusercontent.com/16202812/213757523-c0d34873-50d3-4e19-a0f3-fb0154719e78.png)

I can definitely go into the code and make the changes myself, but I am wondering why this is considered a feature if it doesn't seem to work as intended. If there are any catches or caveats that I missed, please let me know.

**Comment by felixranesberger** [comment]
Same here 👍

**Comment by bechnokid** [comment, marked as answer]
It doesn't look like a solution has been found, so I provided a small workaround.

https://github.com/biati-digital/glightbox/blob/32087845a3872bc875139af9a0a66ae6f92b2240/dist/js/glightbox.js#L1766-L1772

Replace `img` with `slideMedia`. After applying this small change, I was able to adjust the width of the image in a slide. Height works a bit differently and requires some more work, but I think this is a good start.

Closing this out.
